src/app/util/attribute-storage-null-handling.h

```diff
@@ -125,6 +126,46 @@ struct NumericAttributeTraits
     static uint8_t * ToAttributeStoreRepresentation(StorageType & value) { return reinterpret_cast<uint8_t *>(&value); }
 };
 
+template <typename T>
+struct NumericAttributeTraits<BitFlags<T>>
+{
+    using StorageType = T;
+    using WorkingType = BitFlags<T>;
+
+    static constexpr void WorkingToStorage(WorkingType workingValue, StorageType & storageValue) { storageValue = static_cast<StorageType>(workingValue.Raw()); }
+
+    static constexpr WorkingType StorageToWorking(StorageType storageValue) { return WorkingType(storageValue); }
+
+    static constexpr bool IsNullValue(StorageType value) { return value == kNullValue; }
+    static constexpr void SetNull(StorageType & value) { value = kNullValue; }
+    static constexpr bool CanRepresentValue(bool isNullable, StorageType value)
+    {
+        // This treats all nonzero values (except the null value) as true.
+        return !IsNullValue(value);
```

Comment:
This does not seem right.  The right thing should probably be:

1. If !isNullable, return true.
2. If isNullable, return false if the high bit is set.

Reply:
Yeah, will fix. Will add tests (if possible) to validate that.